l10n_it_ddt/views/sale.xml

```diff
@@ -48,8 +48,8 @@
                        name="action_view_ddt"
                        class="oe_stat_button"
                        icon="fa-truck"
-                       attrs="{'invisible': ['|',('ddt_ids','=',False),('ddt_ids','=',[])]}" groups="base.group_user"
-                       string="View DdT">
+                       attrs="{'invisible': ['|',('ddt_ids','=',False),('ddt_ids','=',[])]}" groups="base.group_user">
+                       <field name="ddt_count" widget="statinfo" string="DDT"/>
```

Comment:
```suggestion
                       <field name="ddt_count" widget="statinfo" string="TD"/>
```
Oppure, ancora meglio, lascerei a Odoo il compito di recuperare la stringa corretta dalla definizione del campo.

Reply:
Ho preferito impostare la voce al singolare così come fatto per i [delivery](https://github.com/odoo/odoo/blob/cca4fb071509164493d0febabf53eb43168f6fb6/addons/sale_stock/views/sale_order_views.xml#L16). Per quanto riguarda la stringa in inglese vale quanto scritto in https://github.com/OCA/l10n-italy/pull/1846#discussion_r468576601.